test/data/config.annotations.yaml

```diff
@@ -0,0 +1,2 @@
+- envA/foo: true
```

Comment:
Can you put examples from the issue in here?
https://github.com/screwdriver-cd/screwdriver/issues/580